package.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
It looks like there is a python package now for conventional commits - so maybe we don't need npm/husky here anymore? https://pypi.org/project/conventional-commit/

I don't really mind, but adding it to pre-commit would reduce this dependency

Reply:
Cool, that's worth looking into. It would be nice to remove the node dependency from both Studio and g2p, since it's only for commitlint.

Reply:
Hmm, no, it's an interactive interface to compose commit messages, not a linter. Searching for alternatives, maybe this is a candidate: https://jorisroovers.com/gitlint/